File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikMetricsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.metrics.StepikMetricsPostQuery;

/**
 * @author meanmail
 */
public class StepikMetricsAction extends StepikAbstractAction {
    public StepikMetricsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikMetricsPostQuery post() {
        return new StepikMetricsPostQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikMetricsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.metrics.StepikMetricsPostQuery

class StepikMetricsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun post(): StepikMetricsPostQuery {
        return StepikMetricsPostQuery(this)
    }
}
